apps/designer/app/canvas/shared/use-track-selected-element.ts

```diff
@@ -91,18 +91,18 @@ export const useTrackSelectedElement = () => {
       // It's the second click in a double click.
       if (event.detail === 2) {
         const component = dataset.wsComponent as Instance["component"];
-        if (component === undefined || component in components === false) {
+        if (component === undefined /*|| component in components === false*/) {
```

Comment:
I commented out this temporarily. Forgot to tell you. Should have merge with comment